### PR TITLE
Order emojis by standard ordering, add alternate shortnames

### PR DESCRIPTION
**Pull request by lukebarnard1**
Also, increase the maximum number of emoji shown to 20.

2017-06-27-193735_819x272_scrot

### CODE REVIEW

**Comment by lukebarnard1** [comment]
With `emoji-datasource` we also get `;)` metadata for free! So the next step will be to expand the EMOJI_REGEX and make some adjustments to allow for ";) *tab*" -> :wink:

File: src/autocomplete/EmojiProvider.js [hunk 1]
```diff
@@ -24,10 +24,43 @@ import sdk from '../index';
...
+  'People',
+  'Foods',
+  'Objects',
+  'Activity',
+  'Skin Tones',
+  'Nature',
+  'Places',
+  'Flags',
+  'Symbols',
+];
```
Comment (dbkr) [inline src/autocomplete/EmojiProvider.js]: Should probably fix the indenting here

File: src/autocomplete/EmojiProvider.js [hunk 2]
```diff
@@ -24,10 +24,43 @@ import sdk from '../index';
 import {PillCompletion} from './Components';
 import type {SelectionRange, Completion} from './Autocompleter';
 
+import EmojiData from 'emoji-datasource/emoji';
```
Comment (dbkr) [inline src/autocomplete/EmojiProvider.js]: emoji.json from that package is 874KB - I think this is going to be too big to put in the bundle, realistically

File: src/emoji.json [hunk 3]
```diff
@@ -0,0 +1 @@
...C":{"unified":"270D-1F3FC","image":"270d-1f3fc.png","sheet_x":3,"sheet_y":8,"added_in":"8.0","has_img_apple":true,"has_img_google":true,"has_img_twitter":true,"has_img_emojione":true,"has_img_facebook":true,"has_img_messenger":false},"1F3FD":{"unified":"270D-1F3FD","image":"270d-1f3fd.png","sheet_x":3,"sheet_y":9,"added_in":"8.0","has_img_apple":true,"has_img_google":true,"has_img_twitter":true,"has_img_emojione":true,"has_img_facebook":true,"has_img_messenger":false},"1F3FE":{"unified":"270D-1F3FE","image":"270d-1f3fe.png","sheet_x":3,"sheet_y":10,"added_in":"8.0","has_img_apple":true,"has_img_google":true,"has_img_twitter":true,"has_img_emojione":true,"has_img_facebook":true,"has_img_messenger":false},"1F3FF":{"unified":"270D-1F3FF","image":"270d-1f3ff.png","sheet_x":3,"sheet_y":11,"added_in":"8.0","has_img_apple":true,"has_img_google":true,"has_img_twitter":true,"has_img_emojione":true,"has_img_facebook":true,"has_img_messenger":false}}},{"name":"PENCIL","unified":"270F","variations":["270F-FE0F"],"docomo":"E719","au":"E4A1","softbank":"E301","google":"FE539","image":"270f.png","sheet_x":3,"sheet_y":12,"short_name":"pencil2","short_names":["pencil2"],"text":null,"texts":null,"ca
```
Comment (dbkr) [inline src/emoji.json]: Looks like this file is unused as you import it from the package instead?

Comment (lukebarnard1) [inline src/emoji.json]: Oops, yep this is redundant now.

File: package.json [hunk 4]
```diff
@@ -33,8 +33,9 @@
   "scripts": {
     "reskindex": "node scripts/reskindex.js -h header",
     "reskindex:watch": "node scripts/reskindex.js -h header -w",
-    "build": "npm run reskindex && babel src -d lib --source-maps",
-    "build:watch": "babel src -w -d lib --source-maps",
+    "build": "npm run emoji-data-strip && npm run reskindex && babel src -d lib --source-maps",
```
Comment (dbkr) [inline package.json]: I think you'll need to create the lib directory somewhere, as babel was doing it for us before.

**Pull request merged**